Resources/Prototypes/Catalog/Fills/Crates/service.yml

```diff
@@ -74,3 +74,16 @@
     - id: GroundTobacco
       amount: 4
     - id: Matchbox
+
+- type: entity
+  id: CrateServiceBureaucracy
+  name: bureaucracy crate
+  description: Several stacks of paper and a few pens, what more can you ask for.
+  parent: CrateGenericSteel
```

Comment:
?

Reply:
Is it that the description needs to be lined up better?
Wanted to add this crate in for mimes that lose their paper or wardens that actually write things down or whatever creative uses come up regarding it.

![image](https://user-images.githubusercontent.com/47410468/147414603-216acc1b-e686-40bd-815c-b84f3aa237b3.png)

Reply:
No it's the part where its parent is crategenericsteel.

Reply:
From what I've seen, crategenericsteel allows it to be deconstructed, should it be cratePlastic like the janitorial supplies instead?

Reply:
this is fine?